test/specs/elements/Flag/Flag-test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+import Flag from 'src/elements/Flag/Flag'
+import * as common from 'test/specs/commonTests'
+
+describe('Flag', () => {
+  common.isConformant(Flag)
+
+  it('renders an <i /> element', () => {
+    shallow(<Flag />)
+      .should.have.tagName('i')
+  })
+
+  it('passes property "name" to class', () => {
```

Comment:
Sorry I missed this earlier, this test should be replaced with the common tests for "value only" props:

```js
common.propValueOnlyToClassName(Flag, 'name')
```

This test ensures the prop is defined in _meta, is added to className when it should be, and is not added when it shouldn't be.